src/drivers/mct/shr/seq_timemgr_mod.F90

```diff
@@ -1,8 +1,4 @@
 !===============================================================================
```

Comment:
Looks like this file had a lot of clean-up in addition to the glc-required changes?